packages/amplication-client/src/Profile/accountQueries.ts

```diff
@@ -0,0 +1,25 @@
+import { gql } from "@apollo/client";
+
+export const GET_USER = gql`
+  query getUser {
+    me {
+      account {
+        id
+        email
+        previewAccountEmail
```

Comment:
we'll need to add the previewAccountType as well

Reply:
why?

Reply:
I added it